### DISCUSSION TITLE
Checkpoint Loader with control_after_generate?

### DISCUSSION THREAD

**Discussion by caniyabanci76** [WASasquatch/was-node-suite-comfyui#191] [Q&A]
Hi, was wondering if it is possible to add control_after_generate control for the Checkpoint Loader.  I will often have checkpoints at various step stages of training (i.e. checkpoint_step0100, checkpoint_step0200, checkpoint_step0300 and so on), and wish to create images from them all without having to click around so much. i.e. to have it increment. 

I've been able to kinda do this by making checkpoint name an input and feeding it from a primitive, but if the checkpoints are new, changed, renamed or different, then it doesn't work.

**Comment by WASasquatch** [comment]
I can look into it. Yeah would probably have to draw upon the model paths manually each prompt run. I think the frontend just gets the lost once unless you click the refresh button.

**Reply by caniyabanci76** [reply]
Thanks for the reply.  If it can be done it would be a nice quality-of-life feature!
